src/SourceCode.Clay.OpenApi/Builders/MediaTypeBuilder.cs

```diff
@@ -39,6 +42,8 @@ public class MediaTypeBuilder
         /// </summary>
         public MediaTypeBuilder()
         {
+            Examples = new Dictionary<string, Referable<Example>>();
```

Comment:
StringComparer.Ordinal

Reply:
I will have to write a comparer based on the spec. This will be updated as part of #80.